### PR TITLE
overlord,daemon: another round of typos

### PR DESCRIPTION
**Pull request by zyga**

### CODE REVIEW

File: daemon/daemon.go [hunk 1]
```diff
@@ -434,7 +434,7 @@ var (
 	rebootNoticeWait       = 3 * time.Second
 	rebootWaitTimeout      = 10 * time.Minute
 	rebootRetryWaitTimeout = 5 * time.Minute
-	rebootMaxTentatives    = 3
+	rebootMaxTentative     = 3
```
Comment (MiguelPires) [inline daemon/daemon.go:437]: I think using "tentative" here is still wrong, "attempt" is closer to the intention
```suggestion
	rebootMaxAttempts     = 3
```

File: daemon/daemon.go [hunk 2]
```diff
@@ -682,17 +682,17 @@ func (d *Daemon) RebootDidNotHappen(st *state.State) error {
 		return err
 	}
 	nTentative++
```
Comment (MiguelPires) [inline daemon/daemon.go:684]: Would be nice to change this to attempt as well

File: daemon/daemon.go [hunk 3]
```diff
@@ -682,17 +682,17 @@ func (d *Daemon) RebootDidNotHappen(st *state.State) error {
...
 		restart.ClearReboot(st)
 		clearReboot(st)
-		logger.Noticef("snapd was restarted while a system restart was expected, snapd retried to schedule and waited again for a system restart %d times and is giving up", rebootMaxTentatives)
+		logger.Noticef("snapd was restarted while a system restart was expected, snapd retried to schedule and waited again for a system restart %d times and is giving up", rebootMaxTentative)
 		return nil
 	}
 	st.Set("daemon-system-restart-tentative", nTentative)
 	d.state = st
-	logger.Noticef("snapd was restarted while a system restart was expected, snapd will try to schedule and wait for a system restart again (tenative %d/%d)", nTentative, rebootMaxTentatives)
+	logger.Noticef("snapd was restarted while a system restart was expected, snapd will try to schedule and wait for a system restart again (tenative %d/%d)", nTentative, rebootMaxTentative)
```
Comment (MiguelPires) [inline daemon/daemon.go:695]:
```suggestion
	logger.Noticef("snapd was restarted while a system restart was expected, snapd will try to schedule and wait for a system restart again (attempt %d/%d)", attempt, rebootMaxAttempts)
```

File: daemon/daemon.go [hunk 4]
```diff
@@ -682,17 +682,17 @@ func (d *Daemon) RebootDidNotHappen(st *state.State) error {
...
 	}
 	nTentative++
-	if nTentative > rebootMaxTentatives {
+	if nTentative > rebootMaxTentative {
 		// giving up, proceed normally, some in-progress refresh
 		// might get rolled back!!
 		restart.ClearReboot(st)
 		clearReboot(st)
-		logger.Noticef("snapd was restarted while a system restart was expected, snapd retried to schedule and waited again for a system restart %d times and is giving up", rebootMaxTentatives)
+		logger.Noticef("snapd was restarted while a system restart was expected, snapd retried to schedule and waited again for a system restart %d times and is giving up", rebootMaxTentative)
```
Comment (MiguelPires) [inline daemon/daemon.go:690]:
```suggestion
		logger.Noticef("snapd was restarted while a system restart was expected, snapd retried to schedule and waited again for a system restart %d times and is giving up", rebootMaxAttempts)
```

**Review by MiguelPires — COMMENTED** [review]
thanks, just some suggestions

**Review by ernestl — APPROVED** [review]
Thanks!

File: daemon/daemon.go (hunk 1 above)
Comment (bboozzoo) [reply daemon/daemon.go:437]: sounds like a good idea

**Review by bboozzoo — APPROVED** [review]
typing is hard :smile:

File: daemon/daemon.go (hunk 4 above)
Comment (zyga) [reply daemon/daemon.go]: Done

**Pull request merged**